source/shared/cpp/ObjectModel/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+cmake_minimum_required(VERSION 3.16)
```

Comment:
Are we going to include CMake build process as part of the shared model CI? Is CMake-built shared model binary compatible with the shared model unit tests?

Reply:
I think we will eventually want to stand up a pipeline here, but not as part of this checkin.

The generated binary would theoretically be binary compatible for linking, but we'd have to modify the unit tests. Getting that story sorted out is future work, though :)